shopfloor_reception/models/stock_picking.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    is_shopfloor_return = fields.Boolean()
```

Comment:
And move it to shopfloor

```suggestion
    is_shopfloor_create = fields.Boolean()
```

Reply:
if you want to rename a field is better to do it before merging

Reply:
Fixed in https://github.com/OCA/wms/pull/658
Idd, would be better to integrate here before merging

Reply:
cherry-picked here, so resolving this